packages/dev/core/src/Shaders/ShadersInclude/pbrDebug.fx

```diff
@@ -128,6 +128,14 @@ if (vClipSpacePosition.x / vClipSpacePosition.w >= vDebugMode.x) {
         gl_FragColor.rgb = subSurfaceOut.transmittance;
     #elif DEBUGMODE == 70 && defined(SUBSURFACE) && defined(SS_REFRACTION)
         gl_FragColor.rgb = subSurfaceOut.refractionTransmittance;
+    #elif DEBUGMODE == 72
+        gl_FragColor.rgb = vec3(1.0-roughness); // TODO

```

Comment:
you can replace with `gl_FragColor.rgb = vec3(microSurface);` you can find it here: https://github.com/BabylonJS/Babylon.js/blob/841bdc70820fe8c26f369e60964c7dac8543669d/packages/dev/core/src/Shaders/ShadersInclude/pbrBlockReflectivity.fx

Reply:
Replaced!